### PR TITLE
docs: Edit migrating data with mirrormaker2

### LINKED ISSUE #743: Add large message size to mirrormaker 2 doc
### Describe the Issue

Kafka limits you to 1MB max message size which is frustrating for many kafka users. Redpanda doesn't impose such low limits which makes people very happy. However, when replicating those larger message sizes with mirrormaker 2, they are blocked with an error like `org.apache.kafka.common.errors.RecordTooLargeException: The message is xxxx bytes when serialized which is larger than 1048576, which is the value of the max.request.size configuration.`. While searching for this in Google I couldn't find any good answers on how to configure this, within a couple pages of results I was already starting to read through chinese language web pages to suss out configs. 

### Updates to existing documentation

It is a strong opportunity for us to document it briefly on our "Migrating to Redpanda" docs page *with the exact error message as noted above* so it comes up very quickly on search results. We should point out that Redpanda is more lenient on message sizes and therefore it is common to run into this issue.

This is the configuration needed in mm2.properties if you want to allow 30MB messages, *working as per the configuration we provide on the doc page*:
```redpanda2.producer.max.request.size=31457280```

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves #743 and #673

Preview:
https://deploy-preview-746--redpanda-documentation.netlify.app/docs/data-management/data-migration/

Reviewers, use this opportunity to review the whole page, not just the edits.

### CODE REVIEW

File: docs/data-management/data-migration.mdx [hunk 1]
```diff
@@ -133,11 +134,23 @@ Here are the basic commands to produce and consume streams:
...
+
+## Message size
 
-## Stop MirrorMaker
+Apache Kafka® limits you to 1MB max message size which can be frustrating. Redpanda doesn't impose such low limits. 
 
-To stop the MirrorMaker process, use `top` to find its process ID and then run: `kill <MirrorMaker pid>`
+However, when replicating those larger message sizes with MirrorMaker 2, you can get blocked with an error `org.apache.kafka.common.errors.RecordTooLargeException: The message is xxxx bytes when serialized which is larger than 1048576, which is the value of the max.request.size configuration`. 
+
+To address this issue make sure that your `mm2.properties` configuration file allow up to 30MB messages. This can be done by having this line on the configuration file:
```
Comment (Deflaimun) [inline docs/data-management/data-migration.mdx:149]: @larsenpanda should we include this line on the sample config file as well? 

Also, here should we add a bash command to help the user edit the file?

**Comment by Deflaimun** [comment]
Reviewers, use this opportunity to review the whole page, not just the edits.

File: docs/data-management/data-migration.mdx (hunk 1 above)
Comment (larsenpanda) [reply docs/data-management/data-migration.mdx:149]: I don't think we need to add the bash command to edit the though, personally.

Comment (larsenpanda) [reply docs/data-management/data-migration.mdx:149]: Oh, can you please change to add a note that 30MB is an arbitrary size and they can set it to whatever is needed, as long as the Redpanda cluster itself has not enforced lower sizes.

**Comment by larsenpanda** [comment]
I have one comment on the config, I think we should point out that it's done on the TARGET cluster specifically. "Redpanda2" might be different for their implementation, yes its the syntax in the aforementioned example, but let's just call it out explicitly that it is for the target.

File: docs/data-management/data-migration.mdx (hunk 1 above)
Comment (Deflaimun) [reply docs/data-management/data-migration.mdx:145]: How about

> Apache Kafka® limits you to 1MB max message size which can be frustrating. Redpanda doesn't impose such limits. 

However, when replicating those larger message sizes with MirrorMaker 2, you can get blocked with an error `org.apache.kafka.common.errors.RecordTooLargeException: The message is xxxx bytes when serialized which is larger than 1048576, which is the value of the max.request.size configuration`. 

To address this issue make sure that your `mm2.properties` configuration file allow bigger messages sizes. For example, for 30MB messages make sure that you have this line on the configuration file:

**Comment by Deflaimun** [comment]
@larsenpanda feedback addressed. Please review it and approve the PR if it's correct.

**Review by larsenpanda — APPROVED** [review]

File: docs/platform/data-management/data-migration.mdx [hunk 2]
```diff
@@ -59,42 +60,45 @@ You can find the MirrorMaker script and configuration files in the expanded Kafk
         └── connect-mirror-maker.properties
 ```
 
-The sample configuration is a great place to understand a number of the settings for MirrorMaker.
+The sample configuration is a great place to understand a number of the settings for MirrorMaker 2.
```
Comment (micheleRP) [inline docs/platform/data-management/data-migration.mdx:63]:
```suggestion
The sample configuration describes a number of the settings for MirrorMaker 2.
```

**Comment by micheleRP** [comment]
@Deflaimun: please also search for the typo "seperate" and fix to "separate"

**Review by micheleRP — APPROVED** [review]
Minor style edits, which should be easy to accept + the typo "seperate". After those fixes, looks good!

**Comment by Deflaimun** [comment]
Thanks @micheleRP

**Comment by vuldin** [comment]
@Deflaimun Was this PR supposed to close this ticket? https://github.com/redpanda-data/documentation/issues/673

**Pull request merged**